src/main/java/cubicchunks/worldgen/dependency/DependencyManager.java

```diff
@@ -36,139 +36,92 @@
 import java.util.Map;
 import java.util.Set;
 
+/*
+ * TODO: Abstraction to allow for non-cube dependents.
+ * TODO: Commenting
```

Comment:
The commenting definitely, the non-cube dependents depending on how soon you'd like to merge.